modules/install-osp-deploy-dualstack.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: PROCEDURE
+[id="install-osp-deploy-dualstack_{context}]
+= Deploying the dual-stack cluster
+To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. Following is an example of `install-config.yaml` file.
+
+.Example `install-config.yaml`
+
+[source, yaml]
+----
+apiVersion: v1
+baseDomain: mydomain.test
+featureSet: TechPreviewNoUpgrade <1>
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+metadata:
+  name: mycluster
+networking:
+  machineNetwork: <2>
+  - cidr: "192.168.25.0/24"
+  - cidr: "fd2e:6f44:5dd8:c956::/64"
+  clusterNetwork: <2>
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd01::/48
+    hostPrefix: 64
+  serviceNetwork: <2>
+  - 172.30.0.0/16
+  - fd02::/112
+platform:
+  openstack:
+    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <3>
+    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <3>
+    controlPlanePort:
+      fixedIPs: <4>
+      - subnet:
+          name: subnet-v4 <5>
+          id: subnet-v4-id <5>
+      - subnet:
+          name: subnet-v6 <5>
+          id: subnet-v6-id <5>
+      network:
+        name: dualstack <5>
+        id: network-id <5>
+----
+
+<1> Dual-stack clusters are supported only with `TechPreviewNoUpgrade` value.
+<2> Must have two `cidr` entries. For a cluster with the IPv4 family as the primary address family, specify the IPv4 setting first. For a cluster with the IPv6 family as the primary address family, specify the IPv6 setting first.
+<3> Virtual IP (VIP) address endpoints for the Ingress VIP and API VIP services to provide an interface to the cluster. The sequence of the addresses indicates the primary and secondary VIP address for each service.
```

Comment:
The sequence of the address is not important right now, given we don't support ipv6 primary.
IPv4 being the first is enforced internally by the installer.